site/content/en/contribute/_index.md

```diff
@@ -1,23 +1,8 @@
-
 ---
 title: "Contribute"
```

Comment:
I noticed that we also have a "Community" section in the top bar. Should we get rid of that? It looks like kubernetes.io uses "Community" for this content but our current site uses "Contributing". Personally I like "Contribute" for the content we currently have.

Reply:
agree! great point. let's change it to `Contribute` at the top bar! :)

Reply:
addressed here: https://github.com/kubernetes-sigs/kustomize/pull/4709/commits/8567a233d6308755613ab12b8fb29269fa59ff28